docs/Changelog.md

```diff
@@ -14102,6 +14102,127 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain index tensor to int64</dd>
 </dl>
 
+### <a name="NllLoss-12"></a>**NllLoss-12**</a>
```

Comment:
The generated docs don't say this is operator has a function, because [gen_doc.py](https://github.com/onnx/onnx/blob/master/onnx/defs/gen_doc.py#L204-L207) only checks for `has_function`. It also isn't listed in the table of contents as one of the operators with a function registered.